test/core/GnosisSafe.Setup.spec.ts

```diff
@@ -242,5 +242,13 @@ describe("GnosisSafe", async () => {
 
             await expect(await template.getOwners()).to.be.deep.eq([user1.address, user2.address, user3.address])
         })
+
+        it('should revert if a call to setup modules was made to an eoa account', async () => {
```

Comment:
Nitpick: I would name it `should revert if the initializer address does not contain code.`